templates/aes-agent.yaml

```diff
@@ -18,6 +18,52 @@ metadata:
     product: aes
 ---
 apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: RoleBinding
+metadata:
+  name: {{ include "ambassador.fullname" . }}-agent-config
```

Comment:
Is there any particular reason why we'd want a distinct Rola and RoleBinding for the agent-config?  I kinda expected it to be mashed into a single Role.

Reply:
because then the permissions are confined to the namespace, so we're not giving the agent access to all configmaps in the cluster